app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> object:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: int,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> object:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.wash_single_car(car)
+
+        return income
+
+    def calculate_washing_price(self, car: Car) -> float:
+        clean_mark_diff = self.clean_power - car.clean_mark
+        divident = car.comfort_class * clean_mark_diff * self.average_rating
+        divider = self.distance_from_city_center
+        return round(divident / divider, 1)
+
+    def wash_single_car(self, car: Car) -> float:
```

Comment:
You need to check if clean_power is greater than clean_mark here because if you run this method separately from others it will not work correctly

Reply:
No need to use return here

Reply:
But I need to return the cost of washing the car

Reply:
yep, calculate_washing_price u can use in sevre_cars
and wash_single_car should return nothing

Reply:
![image](https://github.com/mate-academy/py-car-wash-station/assets/80958128/0a6ec415-fb71-46bf-94a2-1352a9c45117)
and wash_single_car u should rewrite like this